pootle/core/search/backends/elasticsearch.py

```diff
@@ -20,47 +23,47 @@
 from ..base import SearchBackend
 
 
+logger = logging.getLogger(__name__)
+
+
 class ElasticSearchBackend(SearchBackend):
     def __init__(self, config_name):
         super(ElasticSearchBackend, self).__init__(config_name)
         self._es = self._get_es_server()
         self._create_index_if_missing()
-        if self._es is not None:
-            self.weight = min(max(self._settings.get('WEIGHT', self.weight),
-                                  0.0), 1.0)
-
-    def _server_setup_and_alive(self):
-        if self._es is None:
-            return False
-        try:
-            return self._es.ping()
-        except ConnectionError:
-            return False
+        self.weight = min(max(self._settings.get('WEIGHT', self.weight),
+                              0.0), 1.0)
 
     def _get_es_server(self):
-        if self._settings is None or Elasticsearch is None:
-            return None
```

Comment:
@unho please see latest commits in #4185 - any misconfigured TM servers are removed by checks